blueprints/controllers/ikea_e2001_e2002/ikea_e2001_e2002.yaml

```
@@ -310,16 +313,16 @@ variables:
     zha:
       button_left_short: [press_257_13_0]
       button_left_long: [hold_3329_0]
-      button_left_release: [release_1365]
+      button_left_release: [release]
       button_right_short: [press_256_13_0]
       button_right_long: [hold_3328_0]
-      button_right_release: [release_-27903]
+      button_right_release: [release]
       button_up_short: ['on']
-      button_up_long: [move_with_on_off_0_83]
-      button_up_release: [stop]
+      button_up_long: [move_with_on_off_MoveMode.Up_83]
+      button_up_release: [stop_with_on_off]
       button_down_short: ['off']
-      button_down_long: [move_1_83]
-      button_down_release: [stop]
+      button_down_long: [move_MoveMode.Down_83_bitmap8.0_bitmap8.0]
+      button_down_release: [stop_with_on_off]
```

```suggestion
      button_down_long: [move_MoveMode.Down_83_0_0]
      button_down_release: [stop_0_0]
```
Based on my testing on HA 2023.6.3, with ZHA, and an E2002 STYRBAR

---

Hi all, @LewisSpring I can commit the change as suggested but I will not be able to test it myself for a while until I have a rpi to test the change myself. For now I will create a new branch on my repository named AHoran/2023.6.3 based on the LordSushiPhoenix/2023.5.x. Please check it one of this days to confirm the change. Thank you.

---

Don't stress, my ego will survive.

---

@LewisSpring I already made the new branch and added the changes on my fork. I still have HA 2023.4.6 and I can test the changes but I'm not expecting it to work. It may be possible but for sure it will take some work to make it compatible with the latest changes.

---

@LewisSpring I don't know how HA updates or not the strings but if they change them the blueprint will not work as expected. I tested the string change for the button_down_long and button_down_release and as I expect, the behavior changed and doesn't work with my current HA version.

---

For reference my quirk is  zhaquirks.ikea.fourbtnremote.IkeaTradfriRemoteV2

And the firmware on the button is Firmware: 0x02040005

It's a newly purchased button in a new HA install.